CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+2. Always make sure you are running the latest revision of the core and database. If you are running something old chances are that your
+bug might already be fixed in the newer revisions.
+
+3. Never post more than **one** bug per issue report. For example; Quest A should have 1 report and Quest B should have another report unless
+they are directly related to each other. (Same goes for Creatures/Raids/Dungeons/Etc.)
+
+4. Never post a copy paste of a crash log in an issue.
+Use a service like [Github Gist](https://gist.github.com/) to paste your crash log and then link it in the issue with a description.
+
+5. If you have trouble writing in English you can use tools like Google Translate to translate from your native language, make sure to always add the original text in your native language (English + Native Language) as well since maybe someone else speaks that language as well.
+
+**Your title should always be after what the issue is about. For example "Quest: The Denouncement" or "Creature: The Lich King"**
+
+**Your issue should always contain information on How it is Working, How it should Work and the revision you are having on your core/DB.**
+
+When you are creating an issue make sure you are following the issue templates, that is important so we can get a report that is clean
+and easy to understand. (issue templates are not made yet).
+
+If you have a fix to your issue please open the issue and then a PR containing the SQL/C++ fix you have so that others can
+review and add onto it. If you do not have a fix for your bug, then just open an issue.
+
+If you are linking SQL data in your report please use a code box with SQL format.
+
+## Opening a PR 📝💻
+
+A Pull Request should always have an opened issue that it refers to.
+
+When opening a PR you should:
+1. Fork NovusCore.
+2. Create a new Branch.
+3. Commit your changes.
+4. Push your changes to your branch.
+5. Open a PR.
+
+You should also always follow the Pull Request Template when opening a PR, that is important so we can get a PR that is clean
+and easy to understand. (Pull Request Template is not made yet).
+
+Before a Pull Request is merged it will go through a couple of checks to makes sure the fix works and is compatible.
+- The PR will be built by the CI's.
+- The PR will be reviewed and approved by someone else on NovusCore to make sure everything is looking okay.
+- The PR will need to be tested in-game to see if it actually a valid fix.
+
+## Feature Requests
+
+All Feature Requests should be opened as an issue with the template for Feature Requests.
+
+If there is a fix for the feature request you can go ahead and open a pull request for it, however, feature requests are only merged if they
+are deemed to be necessary and useful.
+
+## Additional information
+
+We of course appreaciate all contributions you do, small or big. It helps us noticing issues faster and being able to fix them in a good
```

Comment:
We of course appreciate all contributions. It helps us build a better environment for all developers.